### DISCUSSION TITLE
CSS errors

### DISCUSSION THREAD

**Discussion by wirmachenbunt** [cables-gl/cables_docs#271] [Answers & Questions] [answered]
most of example codepen css i copy into the css op throws errors, any idea why ?

![image](https://user-images.githubusercontent.com/1138673/173306026-90158e61-0a7e-4d19-8bc8-61959cad15c3.png)

**Comment by steam0r** [comment, marked as answer]
hey, thats "just" the editor being old/confused and can be ignored

**Reply by wirmachenbunt** [reply]
ja, ok...it also confuses me. i'm not so fluent in css and i don't know what line makes sense and which line doesn't

**Reply by steam0r** [reply]
undestandable, maybe a newer version of the editor has some more modern rules, updating that won't be a quick fix though...

**Reply by wirmachenbunt** [reply]
i better do my css testing outside of cables for now.

and i can imagine, changing the underlying editor stuff is not easy.

**Reply by steam0r** [reply]
was a bit easier than we thought, so we updated the editor to the latest version on dev.cables.gl (cleaning cache before trying out might be a good idea, just to be sure).

would be nice if you could give feedback (feel free to contact me on discord) if things got better for you. maybe we can even go through some of this together.
